unifiedWSHandler.js

```diff
@@ -38,75 +38,68 @@ const environmentToPlatform = {
-                                messageHandler.send(resChann, {embeds: [sendEmbed]}, "Log event")
-                                .then(messageId => {
-                                    if(messageId != -1 && row.autodelete == true){
-                                        const in5minutes = new Date((new Date()).getTime() + 300000);
-                                        pgClient.query("INSERT INTO toDelete (channel, messageID, timeToDelete) VALUES ($1, $2, $3)", [row.channel, messageId, in5minutes])
-                                            .catch(err => {console.log(err);});
-                                    }
-                                });
-                                
-                            }
-                            else{
-                                subscriptions.unsubscribeAll(pgClient, row.channel);
-                                console.log('Unsubscribed from '+row.channel);
-                            } 
-                        }
-                        else{ // DM
-                            messageHandler.send(resChann, {embeds: [sendEmbed]}, "Log event")
-                            .then(messageId => {
-                                if(messageId != -1 && row.autodelete === true){
-                                    const in5minutes = new Date((new Date()).getTime() + 30000);
-                                    pgClient.query("INSERT INTO toDelete (channel, messageID, timeToDelete) VALUES ($1, $2, $3)", [row.channel, messageId, in5minutes])
-                                        .catch(err => {console.log(err);});
-                                }
-                                else if(messageId == -1){
-                                    subscriptions.unsubscribeAll(pgClient, row.channel);
-                                    console.log('Unsubscribed from '+row.channel);
-                                }
-                            })
-                        }                        
-                    })
-                    .catch(error => {
-                        if(typeof(error.code) !== 'undefined'){
-                            if(error.code == 10003){ //Unknown channel error, thrown when the channel is deleted
-                                subscriptions.unsubscribeAll(pgClient, row.channel);
-                                console.log('Unsubscribed from '+row.channel);
-                            }
-                        }
-                        else{
-                            console.log(error);
-                        }
-                    });
+        catch (error) {
+            if (error.code === undefined) {
+                console.log(error);
+            }
+            else if(error.code == 10003){ //Unknown channel error, thrown when the channel is deleted
+                subscriptions.unsubscribeAll(pgClient, row.channel);
+                console.log(`Unsubscribed from ${row.channel}`);
             }
```

Comment:
Add additional logging in case a different error code is thrown?

Reply:
https://github.com/RemainNA/auraxis-bot/pull/64/commits/f7863708550e8c8941e7c904441137b496a722ee